### DISCUSSION TITLE
Should we release 0.67.1?

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#10] [Patches]
# Should we release 0.67.1? 

## 0.67.1 has been released. See discussion for [0.67.2](https://github.com/reactwg/react-native-releases/discussions/12)

Current release: [0.67.0](https://github.com/facebook/react-native/releases/tag/v0.67.0)

Conversations on this thread are limited: 
* [major release issues](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking)
* [qualified cherry-pick requests](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://github.com/facebook/react-native/wiki/Release-FAQ#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue. 
If the issue is a [major release issue](https://github.com/facebook/react-native/wiki/Release-FAQ#what-is-release-blocking), please reference the issue here.

---
#### List of qualified picks
1. Remove alert's window when call to `hide`. ([a46a99e120](https://github.com/facebook/react-native/commit/a46a99e12039c2b92651af1996489d660e237f1b) by [@asafkorem](https://github.com/asafkorem))
2. Do not remove libjscexecutor.so from release builds ([574a773f8f](https://github.com/facebook/react-native/commit/574a773f8f55fe7808fbb672066be8174c64d76d) by [@cortinico](https://github.com/cortinico))

#### Local commits to backport to main
1.

**Comment by numandev1** [comment]
in `0.67.0`, we are using codegen@0.0.8 but for fabric and turbomodule, the recommended version is 0.0.12, 
so please pick https://github.com/facebook/react-native/commit/74f39131283ddd86f458eee9b236b2951fc5e808

**Reply by kelset** [reply]
what do you mean with recommended version? 🤔

**Reply by numandev1** [reply]
https://github.com/facebook/react-native-website/pull/2879/files I was talking about the playbook, Currently, New architecture is published in the nightly releases.

**Reply by kelset** [reply]
@cortinico - what do you think?

**Reply by cortinico** [reply]
I think that cherry-picking would be a "nice-to-have" for a potential 0.67.1 but it's **not**  raising the need for a point release IMHO.

@nomi9995 you're correct to call out that the recommended version to use for `react-native-codegen` is essentially the latest published. On the other hand, the Playbook currently relies on nightly versions of RN.

Moreover, we're leveraging the NPM semantic versioning system meaning that having `"^0.0.11"` as version specified will effectively fetch the **latest minor version** for you (in this case being `0.0.12`).

EDIT: typo

**Reply by tido64** [reply]
> Moreover, we're leveraging the NPM semantic versioning system meaning that having `"^0.0.11"` as version specified will effectively fetch the **latest minor version** for you (in this case being `0.0.12`).

This is a common misconception. ^0.0.11 is actually equivalent to 0.0.11. Semver drop the leading zeroes. You can check it yourself here: https://semver.npmjs.com/

Regardless, agreed that bumping react-native-codegen does not warrant a .1 release.

**Reply by cortinico** [reply]
> Semver drop the leading zeroes

TIL :O Thanks for clarifying @tido64 👍

**Comment by asafkorem** [comment]
I suggest to cherrypick https://github.com/facebook/react-native/commit/a46a99e12039c2b92651af1996489d660e237f1b, this fixes a serious bug with UI alerts on iOS.

**Reply by lunaleaps** [reply]
Picking

**Comment by kelset** [comment]
We are aware of an insta-crash problem when building Android release builds and NOT using Hermes: https://github.com/facebook/react-native/issues/32928 - a PR with a fix is already on its way, will post the commit once it's ready.

**Reply by cortinico** [reply]
The commit is live https://github.com/facebook/react-native/commit/574a773f8f55fe7808fbb672066be8174c64d76d

**Reply by lunaleaps** [reply]
Picking

**Comment by kelset** [comment]
nit: we need to remove from changelog this entry https://github.com/facebook/react-native/commit/6e903b07fa8e8d9b78cae0e031bb8022f7a63195 from here https://github.com/facebook/react-native/blob/main/CHANGELOG.md#ios-specific since it was reverted https://github.com/facebook/react-native/commit/8595f3f22c31e245b7b6b34ec8de011534803a74 - we can do that when doing the changelog PR for 0.67.1
